etl/steps/data/garden/who/2023-07-13/autopsy.meta.yml

```diff
@@ -0,0 +1,41 @@
+dataset:
+  title: European Health Information Gateway (WHO, 2022)
+  description: >-
```

Comment:
To avoid repeating text, you could use here:
`  description:  &dataset-description >-`